Configuration/src/main/java/io/deephaven/configuration/ConfigDir.java

```diff
@@ -52,11 +49,12 @@ public static Path getOrSet(String defaultValue) {
      *
      * @return the configuration file
      */
-    public static String configurationFile() {
-        return Optional
-                .ofNullable(System.getProperty(ROOT_FILE_PROP))
-                .or(ConfigDir::configDirectoryFileIfExists)
-                .orElse(DEFAULT_CONFIGURATION_FILE);
+    public static String defaultConfigurationFile() {
```

Comment:
Why change the name of this? I understand in the context of `ConfigurationContext` that DHE may need to do something different to get a configuration file (ie, not rely on `ConfigDir`?), but in the context of `ConfigDir`, I think `configurationFile()` is a better name?

Reply:
Moreover, it is not returning the `DEFAULT_CONFIGURATION_FILE` except as a last resort, so this does indeed feel like a bad name.

Reply:
This was leftover from a previous aborted attempt. will revert.